webview/src/test/boundingBoxesFixture.ts

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+import {
+  ComparisonBoundingBoxLabels,
+  ComparisonPlotBoundingBoxes,
+  ComparisonRevisionData,
+  PlotsComparisonData
+} from 'dvc/src/plots/webview/contract'
+
+const boxColors = [
```

Comment:
Looking at other `yolo` and other bounding box examples, they seem to rely on a variation of a [Tableau color palette](https://matplotlib.org/stable/gallery/color/named_colors.html#tableau-palette). I've grabbed the set of colors that `yolo` uses for the boxes for now.